gulp/config.js

```diff
@@ -194,48 +195,7 @@ module.exports = (function() {
   // task images: Image build options
   config.images = {
     src: imageFiles,
-    build: build + 'images',
-    minify: true,
-    options: {
-      optimizationLevel: 5,
-      progressive: true,
-      interlaced: true
-    }
-  };
-
-  config.skinImages = {
-    src: [
-      client + 'skin/images/**/*.*'
-    ],
-    build: build + 'images',
-    minify: true,
-    options: {
-      optimizationLevel: 5,
-      progressive: true,
-      interlaced: true
-    }
-  };
-
-
-  config.devImages = {
-    src: imageFiles,
     build: temp + 'images',
-    minify: false
-  };
-
-  config.devSkinImages = {
-    src: [
-      client + 'skin/images/**/*.*'
-    ],
-    build: temp + 'images',
-    minify: false
-  };
-
-  config.imgs = {
```

Comment:
I don't know that we ever actually use any of these, but including the contents of `.tmp/img` does appear to be an omission from the refactored build config...